packages/react/src/components/UIShell/SideNavLink.js

```diff
@@ -16,11 +16,11 @@ import { usePrefix } from '../../internal/usePrefix';
 
 const SideNavLink = React.forwardRef(function SideNavLink(
   {
-    className: customClassName,
     children,
+    className: customClassName,
     renderIcon: IconElement,
     isActive,
-    large,
+    large = false,
```

Comment:
Do we also need to provide the `element = a` default here as well since it was removed with `defaultProps`?

Reply:
@tay1orjones hey we removed it because it was breaking the props. But I believe its because element isn't a prop (I dont know if it was perhaps removed) but since its building from the `Link` component its already an `<a>` element.